helix-rest/src/main/java/org/apache/helix/rest/server/HelixRestServer.java

```diff
@@ -283,7 +283,8 @@ private void cleanupResourceConfigs() {
     }
   }
 
-  public void setupSslServer(int port, SslContextFactory sslContextFactory) {
+
+  public void setupSslServer(int port, SslContextFactory.Server sslContextFactory) {
```

Comment:
Neat : If Now parent factory class is deprecated and directly server and client class are encouraged to access then should we rename this variable as well to "sslContextServer"? Also may be update documentation/comments here?